### PR TITLE
Add JS and Wasm for 2.0.x version

### PR DESCRIPTION
**Pull request by zoobestik**
Fix issues:
✅ Fix for passing `data-target-platform=js-ir`
✅ Accept JS_IR for 2.0.x

Chore:
⚠️ update playwright deps.

Add tests:
➕ Platforms restrictions › JS_IR for unsupported version
➕ Platforms restrictions › JS_IR for supported by *minor* version
➕ Platforms restrictions › JS_IR for supported by *major* version
➕ Platforms restrictions › WASM for unsupported version
➕ Platforms restrictions › WASM for supported by *minor* version [**chromium only**]
➕ Platforms restrictions › WASM for supported by *major* version [**chromium only**]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@babel/plugin-transform-runtime": "~7.22.4",
     "@babel/preset-env": "~7.22.4",
     "@babel/runtime-corejs2": "~7.22.2",
-    "@playwright/test": "^1.37.1",
+    "@playwright/test": "^1.40.1",
```
Comment (zoobestik) [inline package.json:23]: It's required for Wasm test in chrome with enabled WasmGC.

File: src/lib/crosslink.ts [hunk 2]
```diff
@@ -34,11 +33,8 @@ export function generateCrosslink(code: string, options?: LinkOptions) {
 
   if (options && options.targetPlatform) {
     const target =
-      options.targetPlatform && options.targetPlatform.toUpperCase();
-
-    if (!isKeyOfObject(target, TargetPlatforms))
-      throw new Error('Invalid target platform');
-
+      options.targetPlatform && getTargetById(options.targetPlatform);
```
Comment (zoobestik) [inline src/lib/crosslink.ts:36]: It's real bug - it's impossible to pass `js-ir` platform. Cause expected `js_ir`.
Test below.

File: src/webdemo-api.js [hunk 3]
```diff
@@ -50,27 +50,26 @@ export default class WebDemoApi {
    * @returns {*|PromiseLike<T>|Promise<T>}
    */
   static translateKotlinToJs(code, compilerVersion, platform, args, hiddenDependencies) {
-    const MINIMAL_MINOR_VERSION_IR = 5
-    const MINIMAL_MINOR_VERSION_WASM = 9
-    const minor = parseInt(compilerVersion.split(".")[1]);
+    const MINIMAL_MINOR_VERSION_IR = '1.5.0';
+    const MINIMAL_MINOR_VERSION_WASM = '1.9.0';
 
-    if (platform === TargetPlatforms.JS_IR && minor < MINIMAL_MINOR_VERSION_IR) {
```
Comment (zoobestik) [inline src/webdemo-api.js:57]: Main issue `2.0.0` has **minor** less then 5 or 9.

File: tests/utlis/mocks/compiler.ts [hunk 4]
```diff
@@ -30,7 +30,9 @@ function isRunRequest(url: URL | string) {
 
   return (
     uri.host === API_HOST &&
-    uri.pathname.match(/^\/?\/api\/\d+\.\d+\.\d+\/compiler\/run$/) !== null
+    (uri.pathname.match(/^\/?\/api\/\d+\.\d+\.\d+\/compiler\/run$/) !== null ||
+      uri.pathname.match(/^\/?\/api\/\d+\.\d+\.\d+\/compiler\/translate$/) !==
```
Comment (zoobestik) [inline tests/utlis/mocks/compiler.ts:34]: Catch for both types of targets.

**Review by nikpachoo — APPROVED** [review]

**Pull request merged**